src/conn/conn_api.c

```diff
@@ -1022,6 +1022,112 @@ err:	API_END_RET_NOTFOUND_MAP(session, ret);
 }
 
 /*
+ * __conn_optrack_dir --
+ *	Set the directory for operation logging
+ */
+static int
+__conn_optrack_dir(WT_SESSION_IMPL *session, const char *cfg[])
+{
+	WT_CONFIG_ITEM cval;
+
+	/* Only use the environment variable if configured. */
+	WT_RET(__wt_config_gets(session, cfg, "use_environment", &cval));
```

Comment:
I'm inclined to change this, so the configuration option becomes `optrack=(path=,enabled=false)`. We went through an exercise a while ago of avoiding allowing WiredTiger users to create files in arbitrary locations on the system for security reasons, so using a `path` configuration option that has to be a subdirectory of the database is better here.
If a user wants to map that onto a different device then they have the option of pre-creating the directory as a symbolic link.